src/compact.rs

```diff
@@ -192,14 +192,12 @@ impl<T> core::fmt::Debug for Compact<T> where T: core::fmt::Debug {
 	}
 }
 
-#[cfg(feature = "std")]
 impl<T> serde::Serialize for Compact<T> where T: serde::Serialize {
 	fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error> where S: serde::Serializer {
 		T::serialize(&self.0, serializer)
 	}
 }
 
-#[cfg(feature = "std")]
 impl<'de, T> serde::Deserialize<'de> for Compact<T> where T: serde::Deserialize<'de> {
```

Comment:
```suggestion
#[cfg(feature = "serde")]
impl<'de, T> serde::Deserialize<'de> for Compact<T> where T: serde::Deserialize<'de> {
```

Reply:
Doesn't this imply we need to have a `serde` feature under the `[features]` section in `Cargo.toml`?

Reply:
It does.

Reply:
Optional deps are automatically a feature